app/build.gradle

```diff
@@ -460,13 +464,10 @@ task downloadPlugins(
             into 'plugins/crate-azure-discovery'
         }
 
-        // copy crate-admin plugin
         copy {
-            from(project(':admin-ui').buildDir.path + '/crate-admin')
-            from(project(':admin-ui').buildDir.path + '/libs')
-            into 'plugins/crate-admin'
+            from("${buildDir}/crate-admin/")
+            into 'adminui'
```

Comment:
This is a breaking packaging change. I'm not sure if we really want to have a `adminui` folder in the "crate root directory". But it cannot be in the plugins folder any longer as it's missing a plugin description.

Any input here?